torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -350,7 +350,7 @@ std::vector<xla::XlaOp> CreateKthValue(xla::XlaOp input, xla::int64_t k,
 
 std::vector<xla::XlaOp> CreateTopK(xla::XlaOp input, xla::int64_t k,
                                    xla::int64_t dim, bool largest,
-                                   bool /* sorted */) {
+                                   bool /* sorted */, bool stable) {
```

Comment:
I think we should just remove `bool /* sorted */` since it is not being used anyway